package.json

```diff
@@ -15,47 +15,47 @@
     "deep-equal": "1.0.1",
     "fuse.js": "^3.2.0",
     "intl": "1.2.5",
-    "jail-monkey": "0.2.0",
+    "jail-monkey": "1.0.0",
     "mattermost-redux": "mattermost/mattermost-redux#c2cdd8fad33b97af4d0af2462b24f9cfd50a1913",
     "prop-types": "15.6.1",
-    "react": "16.3.1",
+    "react": "16.3.2",
     "react-intl": "2.4.0",
-    "react-native": "0.52.3",
+    "react-native": "0.55.3",
     "react-native-animatable": "1.2.4",
     "react-native-bottom-sheet": "1.0.3",
     "react-native-button": "2.3.0",
-    "react-native-circular-progress": "0.1.0",
+    "react-native-circular-progress": "0.2.0",
     "react-native-cookies": "3.2.0",
-    "react-native-device-info": "0.14.0",
+    "react-native-device-info": "0.21.5",
     "react-native-doc-viewer": "2.7.8",
     "react-native-drawer": "2.5.0",
-    "react-native-exception-handler": "2.7.1",
-    "react-native-fast-image": "4.0.0",
+    "react-native-exception-handler": "2.7.5",
+    "react-native-fast-image": "4.0.8",
     "react-native-fetch-blob": "enahum/react-native-fetch-blob.git#75d5abfa1886665d7eaa947e70b9297b981f0983",
     "react-native-image-gallery": "enahum/react-native-image-gallery#7dd88b037cbb7ee36866585357b33bddb20bd12e",
     "react-native-image-picker": "0.26.7",
     "react-native-keyboard-aware-scroll-view": "0.5.0",
     "react-native-linear-gradient": "2.4.0",
     "react-native-local-auth": "enahum/react-native-local-auth.git#cc9ce2f468fbf7b431dfad3191a31aaa9227a6ab",
-    "react-native-navigation": "1.1.426",
+    "react-native-navigation": "1.1.450",
     "react-native-notifications": "enahum/react-native-notifications.git#663ffe3c4403b185093920c4551f117801499b73",
     "react-native-orientation": "3.1.3",
-    "react-native-passcode-status": "1.1.0",
+    "react-native-passcode-status": "1.1.1",
     "react-native-permissions": "1.1.1",
     "react-native-safe-area": "0.2.3",
     "react-native-section-list-get-item-layout": "2.2.2",
-    "react-native-sentry": "0.35.3",
+    "react-native-sentry": "0.36.0",
     "react-native-slider": "0.11.0",
     "react-native-status-bar-size": "0.3.3",
     "react-native-svg": "6.3.1",
     "react-native-tableview": "2.1.0",
     "react-native-tooltip": "5.2.0",
-    "react-native-vector-icons": "4.5.0",
+    "react-native-vector-icons": "4.6.0",
     "react-native-video": "2.0.0",
     "react-native-youtube": "1.1.0",
-    "react-navigation": "1.5.8",
+    "react-navigation": "1.5.11",
     "react-redux": "5.0.7",
-    "redux": "3.7.2",
+    "redux": "4.0.0",
```

Comment:
Should we do this first with `mattermost-redux`?

Reply:
https://pre-release.mattermost.com/core/pl/3d9gewmjg3y9dei7u1djdzqxac